src/CommunityCommons/javasource/myfirstmodule/actions/Test_Clone_Java.java

```diff
@@ -29,6 +32,8 @@ public java.lang.Boolean executeAction() throws Exception
 		FileDocument doc = new FileDocument(getContext());
 		doc.setName("TestDocument");
 		doc.getMendixObject().setValue(getContext(), "__UUID__", UUID.randomUUID().toString());
+		InputStream is = new ByteArrayInputStream("foo".getBytes(StandardCharsets.UTF_8));
+		doc.getMendixObject().setValue(getContext(), "Contents", is);

```

Comment:
Unlike `__UUID__`, `Contents` is a public attribute. Can you do:
```suggestion
		doc.getMendixObject().setValue(getContext(), FileDocument.MemberNames.Contents.toString(), is);

```

Reply:
Fixed